### PR TITLE
Enable aarch64/ppc64le + more Python versions

### PR DESCRIPTION
**Pull request by leofang**
***Disclaimer: I am not part of the RMM team and this PR is done solely on my own personal capacity being an open source developer and not on behalf of my employer. The expanded builds enabled here are not officially tested by the RMM team.***

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
Please see the disclaimer in the PR description.

**Comment by leofang** [comment]
FYI @alhad-deshpande you need to accept the invite from GitHub so that you can receive notifications from the feedstock (as a maintainer).

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by alhad-deshpande** [comment]
> FYI @alhad-deshpande you need to accept the invite from GitHub so that you can receive notifications from the feedstock (as a maintainer).

Done.

File: conda-forge.yml [hunk 1]
```diff
@@ -1 +1,6 @@
 conda_forge_output_validation: true
+provider:
+  linux_ppc64le: default
+  linux_aarch64: default
```
Comment (leofang) [inline conda-forge.yml:4]: Note: this is the key to tell conda smithy at rerender time that "please consider other non-default platforms".

**Comment by leofang** [comment]
@conda-forge/rmm The CIs are green. PTAL.

**Review by alhad-deshpande — COMMENTED** [review]
Approved

**Comment by leofang** [comment]
@alhad-deshpande I'll let you merge it!

**Comment by alhad-deshpande** [comment]
@leofang 
Still the rmm package is not available on conda-forge. Below search command does not return rmm 21.12.00 package.
conda search rmm -c conda-forge

**Comment by leofang** [comment]
They have been just uploaded to anaconda.org (https://anaconda.org/conda-forge/rmm/files) and it'd take ~2 hrs to sync on CDN so that it's visible to any local conda solvers. Please retry in ~2 hrs 🙂

**Comment by leofang** [comment]
@alhad-deshpande How about now?

**Comment by alhad-deshpande** [comment]
> @alhad-deshpande How about now?

Yes, it is available now and gets installed with following command. 👍 
conda install -c conda-forge rmm=21.12.00 cudatoolkit=11.6

**Comment by leofang** [comment]
Cool, thanks for checking. Good luck with the rest of the package builds!

**Comment by alhad-deshpande** [comment]
> Cool, thanks for checking. Good luck with the rest of the package builds!

Thanks for all your help 👍 You can help me in remaining package builds :)

**Pull request merged**